nicegui/elements/chart.js

```diff
@@ -1,10 +1,21 @@
 export default {
   template: "<div></div>",
   mounted() {
-    this.chart = Highcharts.chart(this.$el, this.options);
+    for (const extra of this.extras) {
+      console.log("loading extra " + extra);
+      import(extra)
+        .then((module) => {
+          console.log("loaded extra " + extra);
+        })
+        .catch((err) => {
+          console.log("error loading extra " + extra);
+          console.log(err);
+        });
+    }
     setTimeout(() => {
+      this.chart = Highcharts.chart(this.$el, this.options);
       this.chart.reflow();
-    }, 0);
+    }, 1000);
```

Comment:
I'm not happy with the delayed loading of the chart. We should somehow make sure that the chart is loaded as soon as the extra libs are available...